### PR TITLE
Retire Packages, Metapackages, and Frameworks

### LINKED ISSUE #1311: Retire topic: Packages, Metapackages and Frameworks
Topic URL: https://docs.microsoft.com/en-us/dotnet/articles/core/packages

Comments on this topic indicate that people are confused and they don't understand the terminology used and the concepts introduced.

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #1311

### CODE REVIEW

File: .openpublishing.redirection.json [hunk 1]
```diff
@@ -348,6 +348,10 @@
       "redirect_url": "/dotnet/core/migration/from-dnx",
       "redirect_document_id": true
     },
+    {
+      "source_path": "/docs/core/packages.md",
+      "redirect_url": "/dotnet/https://docs.microsoft.com/en-us/dotnet/core/deploying"
```
Comment (pkulikov) [inline .openpublishing.redirection.json:353]:
```suggestion
      "redirect_url": "/dotnet/core/deploying"
```

**Comment by tdykstra** [comment]
Thanks @pkulikov!

**Review by CamSoper — APPROVED** [review]

**Comment by mairaw** [comment]
This PR introduced some warnings

**Comment by tdykstra** [comment]
@mairaw Thanks for letting me know.

**Pull request merged**